flink-runtime/src/main/java/org/apache/flink/runtime/highavailability/JobResultStoreOptions.java

```diff
@@ -65,4 +65,11 @@ public class JobResultStoreOptions {
                                     + "are, instead, marked as clean to indicate their state. In this "
                                     + "case, Flink no longer has ownership and the resources need to "
                                     + "be cleaned up by the user.");
+
+    @Documentation.Section(Documentation.Sections.COMMON_HIGH_AVAILABILITY_JOB_RESULT_STORE)
+    public static final ConfigOption<Integer> TIME_TO_REMOVE_CLEAN_JOB_RESULT =
+            ConfigOptions.key("job-result-store.ttl-clean-job-result")
+                    .intType()
```

Comment:
```suggestion
                    .durationType()
```
There is a `durationType()` for config parameters. That allows the user to specify any time unit.